backend/migrations/2023-02-04-132532_create_plant_detail/up.sql

```diff
@@ -0,0 +1,93 @@
+-- Your SQL goes here
+CREATE TYPE WATER AS ENUM ('low', 'moderate', 'high', 'aquatic');
+CREATE TYPE SUN AS ENUM ('indirect sun', 'partial sun', 'full sun');
+CREATE TYPE SHADE AS ENUM (
+  'no shade',
+  'light shade',
+  'partial shade',
+  'permanent shade',
+  'permanent deep shade'
+);
+CREATE TYPE SOIL_PH AS ENUM (
+  'very acid',
+  'acid',
+  'neutral',
+  'alkaline',
+  'very alkaline'
+);
+CREATE TYPE SOIL_TEXTURE AS ENUM ('sandy', 'loamy', 'clay', 'heavy clay');
+CREATE TYPE SOIL_WATER_RETENTION AS ENUM ('well drained', 'moist', 'wet');
+CREATE TYPE LIFE_CYCLE AS ENUM ('annual', 'biennial', 'perennial');
+CREATE TYPE GROWTH_RATE AS ENUM ('slow', 'moderate', 'vigorous');
+CREATE TYPE FLOWER_TYPE AS ENUM ('dioecious', 'monoecious', 'hermaphrodite');
+CREATE TYPE FERTILITY AS ENUM ('self fertile', 'self sterile');
+CREATE TYPE HERBACEOUS_OR_WOODY AS ENUM ('herbaceous', 'woody');
+CREATE TYPE DECIDUOUS_OR_EVERGREEN AS ENUM ('deciduous', 'evergreen');
+CREATE TYPE ROOT_ZONE_TENDANCY AS ENUM ('surface', 'shallow', 'deep');
+CREATE TABLE plant_detail (
+  id SERIAL PRIMARY KEY,
+  binomial_name VARCHAR NOT NULL,
+  common_name TEXT ARRAY,
+  common_name_de TEXT ARRAY,
+  folder_name VARCHAR,
+  genus VARCHAR,
+  family VARCHAR,
+  subfamily VARCHAR,
+  edible_uses VARCHAR,
+  medicinal_uses VARCHAR,
+  material_uses_and_functions VARCHAR,
+  botanic VARCHAR,
+  propagation VARCHAR,
+  cultivation VARCHAR,
+  environment VARCHAR,
+  material_uses VARCHAR,
+  functions VARCHAR,
+  provides_forage_for VARCHAR,
+  provides_shelter_for VARCHAR,
```

Comment:
These are quite unimportant, please move "documentation-like" attributes to end (also in CSV if easily possible)